script.js

```diff
@@ -1,9 +1,13 @@
-var display = document.getElementById("screen");
-var buttons = document.getElementsByClassName("button");
-
-Array.prototype.forEach.call(buttons, function (button) {
+let display = document.getElementById("screen");
+const buttons = Array.from(document.querySelectorAll(".button"));
+const backbtn=document.querySelector('.back');

```

Comment:
if you would use the `button` class (as all other buttons), then you dont need this line.